.changeset/modern-ghosts-sin.md

```diff
@@ -0,0 +1,31 @@
+---
+"@logto/tunnel": minor
+---
+
+add deploy command and env support.
+
+#### Add new `deploy` command to deploy your local custom UI assets to your Logto Cloud tenant
+
+1. Create a Machine-to-Machine app in your Logto tenant
+2. Run the following command
+
+```bash
+npx @logto/tunnel deploy --app-id <your-m2m-app-id> --app-secret <your-m2m-app-secret> --endpoint https://<tenant-id>.logto.app --experience-path /path/to/your/custom/ui
```

Comment:
instead of having two separate options (app id and app secret), will it be better to use one option e.g. `auth` with the basic auth format? in the future we can reuse the same option with different formats

Reply:
Then would it be too cumbersome to use?

Reply:
Offline discussed and we'll use `--auth appId:appSecret` instead